oneandone/resource_oneandone_server.go

```diff
@@ -282,7 +299,15 @@ func resourceOneandOneServerRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("name", server.Name)
 	d.Set("datacenter", server.Datacenter.CountryCode)
 
-	d.Set("hdds", readHdds(server.Hardware))
+	if server.Hardware.FixedInsSizeId != "" {
+		fixedsize, err := Id2fixedsize(config, server.Hardware.FixedInsSizeId)
+		if err != nil {
+			return err
+		}
+		d.Set("fixedsize", fixedsize)
+	} else {
+		d.Set("hdds", readHdds(server.Hardware))
+	}
```

Comment:
have you checked if the server is created correctly ?
if i send this query the server is still created with the default disk-size according to their webinterface.